backend/scheduler/client.go

```diff
@@ -39,12 +42,38 @@ type QueuedItem struct {
 func (s *Service) Queue(ctx context.Context, collection string, id string, at time.Time) error {
 	client, _ := cloudtasks.NewClient(ctx)
 
+	it := client.ListTasks(ctx, &cloudtaskspb.ListTasksRequest{
+		Parent: s.queueID,
+	})
+
 	taskName := s.queueID + "/tasks/" + collection + "-" + id
 
+	for {
+		resp, err := it.Next()
+		if err == iterator.Done {
+			break
+		}
```

Comment:
```suggestion
	for resp, err := it.Next(); err != iterator.Done {
```

I *think* this is valid go code. 
Don't have to use it if you don't like it. I was just looking at the pattern for getting SQL rows, but its iterator operates differently:

```
for rows.Next() {
	err := rows.Scan(&id, &name)
	if err != nil {
		log.Fatal(err)
	}
	log.Println(id, name)
}
```